test/core/tables.wast

```diff
@@ -0,0 +1,14 @@
+;; Duplicate table identifiers
```

Comment:
Nit: rename file to table.wast, to be named after the construct (like similar cases).

Reply:
Ah ok, should `globals.wast` get renamed to `global.wast`?

Reply:
Yeah, it should. It's one of the clean-ups that ought to be done for the current test suite. Feel free to include it if you feel like it. :)